third_party/terraform/website/docs/guides/version_3_upgrade.html.markdown

```diff
@@ -466,12 +467,17 @@ using this field from Feb 1, 2020 onwards.
 `auto_create_routes` has been removed because it's redundant and not
 user-configurable.
 
-## Resource: `google_compute_region_instance_group_manager`
+## Resource: `google_compute_instance_group_manager`
 
-### `update_strategy` no longer has any effect and is removed
+The following changes apply to both `google_compute_instance_group_manager` and `google_compute_region_instance_group_manager`.
 
-With `rolling_update_policy` removed, `update_strategy` has no effect anymore.
-Before updating, remove it from your config.
+### `instance_template` has been replaced by `version.instance_template`
+
+Instance group managers should be using `version` blocks to reference which instance template to use for provisioning. To upgrade use a single `version` block with `instance_template` in your config and by default all traffic will be directed to that version.
+
+### `update_strategy` has been replaced by `update_policy`
+
+To allow much greater control over the updates happening to instance groups `update_strategy` has been replaced by `update_policy`. The previous functionality to determine if instance should be replaced or restarted can be achieved using `update_policy.minimal_action`. For more details see the [official guide.](https://cloud.google.com/compute/docs/instance-groups/rolling-out-updates-to-managed-instance-groups)
```

Comment:
I'd be fine either way but if you want to add old/new config, that might be nice? Also, I'm not sure if we should keep a heading for regional IGM just in case someone is copy pasting against only one